creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -1365,106 +1368,6 @@ class SnapbackSM {
     return { requiredUpdateReplicaSetOps, potentialSyncRequests }
   }
 
-  /**
-   * Used to determine the `requiredUpdateReplicaSetOps` and `potentialSyncRequests` for a given nodeUser, given that the
-   * nodeUser observed did not return any spIds for the replica set from the Discovery query.
-   * @param {Object} nodeUser {secondary1, secondary2, primary, user_id, wallet}
-   * @param {Set<string>} unhealthyPeers set of unhealthy peers
-   * @param {Object[]} requiredUpdateReplicaSetOps array of {...nodeUsers, unhealthyReplicas: {string[]} endpoints of unhealthy rset nodes }
-   * @param {Object[]} potentialSyncRequests array of {...nodeUsers, endpoint: {string} endpoint to sync to }
-   *
-   * @note this is for backwards compatibility. Once all the discovery nodes have upgraded, we can deprecate and remove this method.
-   */
-  async _aggregateOpsWithoutQueriedSpIds(nodeUser, unhealthyPeers) {
```

Comment:
love the red 🔴

Reply:
seems like all discovery nodes are up to date now? hence allowing us to remove this legacy code?

Reply:
yeah! https://dashboard.audius.org/#/services/discovery-node takes a long time to load, but I confirmed that eventually it shows they're all up to date